commands/set.js

```diff
@@ -11,71 +11,70 @@
 // OR the same as:
 // const [action, key, ...value] = args;
 exports.run = async (client, message, [action, key, ...value], level) => { // eslint-disable-line no-unused-vars
-  
 
-  // First we need to retrieve current guild settings
```

Comment:
Not sure why you're changing the language used from warm and friendly to cold and clinical.

Reply:
changed to previous